classes/book.rb

```diff
@@ -0,0 +1,12 @@
+require '../item'
+class Book < Item
+  def initialize(publisher, cover_state, publish_date = Time.year)
+    super(publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?
+    true if Time.now.year - @publish_date.year > 10 || @cover_state == 'bad'
```

Comment:
- [ ] `archived`canbe modified depending on property from a parent class or cover state.
```suggestion
super || cover_state == 'bad'
```

Reply:
I don't think archived is needed, all that is needed for can_be_archived is publish_date, not archived

Reply:
can you explain better what you mean